CodeGeneration/Sources/SyntaxSupport/Node.swift

```diff
@@ -24,6 +24,7 @@ public class Node {
   public let description: String?
   public let baseKind: String
   public let parserFunction: String?
+  public let expressibleByStringLiteral: Bool
```

Comment:
I'm not really sure I see the point of separating these. If we think something is worth having a public parse method for then it's probably also worth being expressible by string interpolation as well. I imagine this bumps up the count of non-base nodes though :(.

Reply:
My initial idea was that we could keep the list of exceptions for non-base nodes really small (<5 let’s say). But I’ve got a feeling that I’m loosing that battle in this PR already. I’ve removed that property again.